src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -0,0 +1,625 @@
+# Dynamic proxy configuration
+
+The dynamic proxy configuration feature allows operations to provide a customized proxy configuration that would be updated automatically according to the applications that are running within ConductR cluster.
+
+ConductR provides a default configuration with can be overridden should the need arises.
```

Comment:
arise. For example, HAProxy can provide HTTP Basic Auth for staging deployments enabling protected preview prior to promoting to production. This is accomplished by deploying a template haproxy.cfg file as part of the configuration of the HAproxy bundle.

This document is a guide and reference for using dynamic HAProxy configuration templates.